### ISSUE TITLE
Speed up ant_metrics

### ISSUE THREAD

**Issue by plaplant** [HERA-Team/hera_qm#113]
The largest bottleneck in RTP currently is the ant_metrics stage of processing. We should make this step take less time to better fit within the "real-time" constraint of the system.

@jsdillon I'm tentatively assigning this to you, but feel free to reassign as necessary. Thanks for looking into this.

**Comment by jsdillon** [comment]
The simplest minor fix is to just throw away antennas that appear really dead (dead_metric > 10) whenever they appear, even if they aren't the worst offenders. This eliminates many of the iterations of ant_metrics, though not all.

I've verified on a day of ant_metrics data that this doesn't throw away any extra antennas that weren't going to get thrown out eventually.

```
import glob
amfiles = glob.glob('../../hera_sandbox/jsd/Antenna_Metrics/sample_jsons/*')
ams = [ant_metrics.load_antenna_metrics(f) for f in amfiles]

starts_below_10 = set([])
starts_above_10 = set([])
ends_above_5 = set([])
ends_below_5 = set([])

for n,am in enumerate(ams):
    for ant in am['final_mod_z_scores']['meanVij'].keys():
        final_dead_metric = np.abs(am['final_mod_z_scores']['meanVij'][ant]/2) + np.abs(am['final_mod_z_scores']['redCorr'][ant]/2)
        starting_dead_metric = np.abs(am['all_mod_z_scores'][0]['meanVij'][ant]/2) + np.abs(am['all_mod_z_scores'][0]['redCorr'][ant]/2)
        if final_dead_metric > 5:
            ends_above_5.add(ant+(n,))
        if final_dead_metric <= 5:
            ends_below_5.add(ant+(n,))
        if starting_dead_metric > 10:
            starts_above_10.add(ant+(n,))
        if starting_dead_metric <= 10:
            starts_below_10.add(ant+(n,))

print 'starts_below_10', len(starts_below_10)
print 'starts_above_10', len(starts_above_10)
print 'ends_below_5', len(ends_below_5)
print 'ends_above_5', len(ends_above_5)

print 'starts above 10 and ends below 5:', len(starts_above_10.intersection(ends_below_5))
print 'starts below 10 and ends above 5:', len(starts_above_10.intersection(ends_above_5))
```
which yields
```
starts_below_10 9000
starts_above_10 216
ends_below_5 8889
ends_above_5 327
starts above 10 and ends below 5: 0
starts below 10 and ends above 5: 216
```

So that should speed things up by a factor of 2-3, which seems good enough for this season.

**Comment by plaplant** [comment]
This was addressed in PR #115.
